### PR TITLE
Update Chromebook steps and recommend extension as workaround

### PR DESCRIPTION
**Pull request by seaxwi**

### CODE REVIEW

File: content/Arduino Cloud/Arduino Chrome App/detect-device-button.png
Comment (per1234) [inline content/Arduino Cloud/Arduino Chrome App/detect-device-button.png:1]: This is an unused copy of the file so it should be deleted. The one in use is at `img/detect-device-button.png`.

File: content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md [hunk 1]
```diff
@@ -65,21 +60,13 @@ Follow these steps:
 
 1. Connect a [compatible board](https://support.arduino.cc/hc/en-us/articles/360014779899#chromebook) to one of the USB ports on your Chromebook.
 
-1. In the board selector menu, choose **Select Other Board & Port**
-
-   ![The Boards Dropdown in the Cloud Editor.](img/web-editor-boards-dropdown.png)
```
Comment (per1234) [inline content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md:70]: `img/web-editor-boards-dropdown.png` is now unused so the file should be removed from the repository.

File: content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md [hunk 2]
```diff
@@ -65,21 +60,13 @@ Follow these steps:
...
-
-1. Pick the specific board you're using and click **OK**.
-
-1. Click the **Upload** button ![Upload button](img/upload_button.png) in the editor.
```
Comment (per1234) [inline content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md:74]: `img/upload_button.png` is now unused so the file should be removed from the repository.

File: content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md [hunk 3]
```diff
@@ -65,21 +60,13 @@ Follow these steps:
...
-
-1. That should open a **Authorise USB connection** window. Click **Select port**.
+1. In the board selector menu, click on the **Detect Device** button.
 
-   ![Authorise USB connection](img/Authorise-USB-connection.png)
```
Comment (per1234) [inline content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md:78]: `img/Authorise-USB-connection.png` is now unused so the file should be removed from the repository.

File: content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md [hunk 4]
```diff
@@ -65,21 +60,13 @@ Follow these steps:
...
+   ![The Detect Device button.](img/detect-device-button.png)
 
-1. A **ChromeOS** window should appear. Select the desired port and click **Connect**.
+1. Select the serial port for your board board you're using and click **Connect**. If you're unsure which serial port to use, try disconnecting and reconnecting your device. The serial port for your board will disappear and reappear in the list.
 
-   ![ChromeOS window](img/usbpromptchrome.png)
```
Comment (per1234) [inline content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md:82]: `img/usbpromptchrome.png` is now unused so the file should be removed from the repository.

**Review by per1234 — CHANGES_REQUESTED** [review]

File: content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md (hunk 2 above)
Comment (seaxwi) [reply content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md:74]: Resolved by 68d8538bd1a45060e958b183f26aa132304e0b89.

File: content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md (hunk 4 above)
Comment (seaxwi) [reply content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md:82]: Resolved by 68d8538bd1a45060e958b183f26aa132304e0b89.

**Pull request merged**